### ISSUE TITLE
How many simultaneous connections can iOS and Android local P2P keep open?

### ISSUE THREAD

**Issue by yaronyg** [thaliproject/Thali_CordovaPlugin#335]
This question is not so much about performance as handle management. The notification layer in particular needs to be able to open lots of connections to lots of people and receive lots of connections from lots of people. But we need to know how many open connections each platform's native stack can handle. This will tell us, for example, if we need to aggressively close connections before we open new ones or if we can just let connections time out when they are not in use.

To be clear this is not actually a performance question. I will file a separate bug on that. It's a technical capacity question. How many handles can each platform keep open before they give up?

In the case of bluetooth one suspects the answer is a function of the combination of connections in each direction due to limitations in the hardware that Bluetooth typically uses.

So basically we just want to open and receive as many connections as possible and see when the stacks fall over.

**Comment by yaronyg** [comment]
I'm assigning this to @tobybrad as a lower priority item that he can assign from the stack if a miracle happens and the group has time to get to it.
